src/components/pages/Sent/index.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+.end
+  h3 {{ $t('sent.thanks', { name: report.owner.name }) }}
+  
+  button(@click="clear") {{ $t('sent.finish') }}
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapGetters({
+      report: 'report'
+    })
+  },
+  beforeMount () {
+    if (this.report.owner.name === '') {
```

Comment:
You should abstract this to an authorization service or something like that that runs every time that you change routes. That way you can keep the component clean of this `beforeMount()` behavior that's not really well suited in here.